src/main/java/RacingCars.java

```diff
@@ -7,16 +7,32 @@
 
   public RacingCars(int carAmount) {
     this.racingCars = new ArrayList(carAmount);
+    createRacingCars(carAmount);
+  }
+
+  private void createRacingCars(int carAmount) {
     for (int i = 0; i < carAmount; i++) {
       this.racingCars.add(new RacingCar());
     }
   }
 
+  public RacingCars(String[] racingCars) {
+    int carAmount = racingCars.length;
+    this.racingCars = new ArrayList(carAmount);
+    createRacingCars(racingCars);
+  }
+
+  private void createRacingCars(String[] racingCars) {
+    for (int i = 0; i < racingCars.length; i++) {
+      this.racingCars.add(new RacingCar(racingCars[i]));
+    }
+  }
```

Comment:
다음과 같이 향상된 for 문을 사용해보면 어떨까요? 😄 
```suggestion
  private void createRacingCars(String[] racingCars) {
    for (String racingCar : racingCars) {
      this.racingCars.add(new RacingCar(racingCar));
    }
  }
```